src/client-entry.ts

```diff
@@ -4,16 +4,26 @@ import buildApp from '@/common/buildApp';
 import TermboxRequest from '@/common/TermboxRequest';
 import { factory } from '@/common/TermboxFactory';
 import UlsLanguageTranslationRepository from '@/client/data-access/UlsLanguageTranslationRepository';
+import UlsLanguageRepository from '@/client/data-access/UlsLanguageRepository';
 import EntityRepository from '@/client/data-access/EntityRepository';
 import MwWindow from '@/client/mediawiki/MwWindow';
 import { Hooks } from '@/client/mediawiki/Hooks';
 
 Vue.config.productionTip = false;
 
 factory.setLanguageTranslationRepository(
-	new UlsLanguageTranslationRepository( new ( window as MwWindow ).wb.WikibaseContentLanguages() ),
+	new UlsLanguageTranslationRepository(
+		new ( window as MwWindow ).wb.WikibaseContentLanguages(),
+	),
 );
-// TODO factory.setLanguageRepository( languageRepo );
+
+factory.setLanguageRepository(
+	new UlsLanguageRepository(
+		new ( window as MwWindow ).wb.WikibaseContentLanguages(),
```

Comment:
If we consider `WikibaseContentLanguages` a service (I do), we should reuse it.

Reply:
agree